docs/source/guide/cdr-1-intro.myst

```diff
@@ -0,0 +1,137 @@
+1. A near-Clifford (classical) circuit simulator.
+
++++
+
+### 1. Define a quantum circuit
+
+The quantum circuit can be specified as any quantum circuit supported by Mitiq but **must be compiled into the gateset $\{ \sqrt{X}, Z, \text{CNOT}\}$.**
+
+```{code-cell} ipython3
+a, b = cirq.LineQubit.range(2)
+circuit = cirq.Circuit(
+    cirq.rx(0.1).on(a),
+    cirq.rx(-0.72).on(b),
+    cirq.rz(0.4).on(a),
+    cirq.rz(0.2).on(b),
+    cirq.CNOT.on(a, b),
+    cirq.rx(-0.1).on(b),
+    cirq.rz(-0.23).on(a),
+    cirq.CNOT.on(b, a),
+    cirq.rx(-0.112).on(a),
+)
+circuit
+```
+
+### 2. Define an executor
+The executor inputs a circuit and returns a {class}`.QuantumResult`. Typically this function will send the circuit to a quantum computer and wait for the results. Here for sake of example we use a simulator that adds single-qubit depolarizing noise after each moment and returns the final density matrix.
+
+```{code-cell} ipython3
+from mitiq.interface.mitiq_cirq import compute_density_matrix
+
+compute_density_matrix(circuit).round(3)
+```
+
+### 3. Observable
+The observables $O$ indicates what we wish to compute via $\text{Tr} [ \rho O ]$.
+
+```{code-cell} ipython3
+# Observable to measure.
+obs = Observable(PauliString("ZZ"), PauliString("X", coeff=-1.75))
+print(obs)
+```
+
+You can read more about this Mitiq object in the [documentation](guide-observables.myst).
+
++++
+
+### 4. (Near-Clifford) Simulator
+The CDR method creates a set of "training circuits" which are related to the input circuit and are efficiently simulable. These circuits are simulated on a classical (noiseless) simulator to collect data for regression. The simulator should also return a `QuantumResult`.
+
+To use CDR at scale, an efficient near-Clifford circuit simulator must be specified. In this example, the circuit is small enough to use any classical simulator, and we use the same density matrix simulator as above but without noise.
+
+```{code-cell} ipython3
+def simulate(circuit: cirq.Circuit) -> np.ndarray:
+    return compute_density_matrix(circuit, noise_level=(0.0,))
+
+simulate(circuit).round(3)
+```
+
+## Run CDR
+
+Now we can run CDR. We first compute the noiseless result then the noisy result to compare to the mitigated result from CDR.
+
++++
+
+### The noiseless result
+
+```{code-cell} ipython3
+ideal_measurement = obs.expectation(circuit, simulate).real
+print("ideal_measurement = ",ideal_measurement)
+```
+
+### The noisy result
+We now generate the noisy result. Note that {func}`.compute_density_matrix` function by default runs a simulation with noise.
+
+```{code-cell} ipython3
+unmitigated_measurement = obs.expectation(circuit, compute_density_matrix).real
+print("unmitigated_measurement = ", unmitigated_measurement)
+```
+
+### The mitigated result
+
+```{code-cell} ipython3
+mitigated_measurement = cdr.execute_with_cdr(
+    circuit,
+    compute_density_matrix,
+    observable=obs,
+    simulator=simulate,
+).real
```

Comment:
```suggestion
mitigated_measurement = cdr.execute_with_cdr(
    circuit,
    compute_density_matrix,
    observable=obs,
    simulator=simulate,
    seed=0,
).real
```

Add a seed to avoid non-deterministic results.